### ISSUE TITLE
NumberFormatException crash

### ISSUE THREAD

**Issue by RicardoBelchior** [VerveWireless/verve-android-adsdk#15]
Hi,

We've been receiving a huge number of the following crash on a small number of users:

```
Fatal Exception: java.lang.NumberFormatException: For input string: "+HY1wdyDdrTTF6m7Zvp5NQ=="
       at java.lang.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:1306)
       at java.lang.Float.parseFloat(Float.java:459)
       at com.vervewireless.advert.d.k.a(SourceFile:89)
       at com.vervewireless.advert.d.au.e(SourceFile:66)
       at com.vervewireless.advert.b.e.a(SourceFile:254)
       at com.vervewireless.advert.b.e.e(SourceFile:272)
       at com.vervewireless.advert.b.e.c(SourceFile:320)
       at com.vervewireless.advert.b.b.b(SourceFile:178)
       at com.vervewireless.advert.b.b$a.onConnected(SourceFile:198)
       at com.google.android.gms.common.internal.zzae.zzk(Unknown Source)
       at com.google.android.gms.common.api.internal.zzba.zzj(Unknown Source)
       at com.google.android.gms.common.api.internal.zzao.zzaie(Unknown Source)
       at com.google.android.gms.common.api.internal.zzao.onConnected(Unknown Source)
       at com.google.android.gms.common.api.internal.zzbi.onConnected(Unknown Source)
       at com.google.android.gms.common.api.internal.zzt.onConnected(Unknown Source)
       at com.google.android.gms.common.internal.zzac.onConnected(Unknown Source)
       at com.google.android.gms.common.internal.zzn.zzakr(Unknown Source)
       at com.google.android.gms.common.internal.zze.zzw(Unknown Source)
       at com.google.android.gms.common.internal.zzi.zzaks(Unknown Source)
       at com.google.android.gms.common.internal.zzh.handleMessage(Unknown Source)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:154)
       at android.app.ActivityThread.main(ActivityThread.java:6823)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1563)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1451)
```

Don't know what it is but looks like a preventable one...

Cheers,
Ricardo

**Comment by christinamatthews** [comment]
@RicardoBelchior - Thank you for your email. We have put a fix in place for this for our next release.

**Comment by christinamatthews** [comment]
Issue resolved in v3.7.2.
